swarm/src/connection/pool.rs

```diff
@@ -307,6 +307,8 @@ where
 {
     /// Creates a new empty `Pool`.
     pub(crate) fn new(local_id: PeerId, config: PoolConfig) -> Self {
+        let span = tracing::error_span!("peer id:", %local_id, connection_id = tracing::field::Empty);
```

Comment:
The way spans work is that they need to be within the stack of all function calls in which logged messages should contain this context. Think about it as "invisible" function parameters.

The `new` function does not have an `tracing::log` statements so this span does not nothing.